apps/devhub/tests/test_models.py

```diff
@@ -205,6 +207,28 @@ def test_tag_no_match(self):
         # There should only be one a, the link to the addon, but no tag link.
         eq_(len(pq(text)('a')), 1)
 
+    def test_beta_signed_events_validation_passed(self):
+        addon = Addon.objects.get()
+        file_ = addon.versions.first().files.first()
+        # Signed beta file which passed validation.
+        amo.log(amo.LOG.BETA_SIGNED_VALIDATION_PASSED, file_)
+        log = ActivityLog.objects.beta_signed_events().first().to_string()
+        link = pq(log)('a')
+        assert len(link) == 1
+        assert link[0].attrib['href'] == reverse('files.list', args=[file_.pk])
+        assert 'passed' in log
```

Comment:
Can you test for the entire expected HTML string here? Stringification for these log entries has a tendency to break in unexpected ways. Same below.